neptune-python-utils/build.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -ex
 
 pushd .
+sudo su 
```

Comment:
Should this be `sudo pip install virtualenv`?

Reply:
Update to your suggestion